docker-for-mac/faqs.md

```diff
@@ -154,6 +154,13 @@ response.
 If you do not want auto-send of usage data, use the Stable channel. For more
 information, see [Stable and Edge channels](#stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker for Mac?").
 
+### How does Docker Desktop handle Personal Data?
+
+When uploading diagnostics to Docker to help issue investigation, diagnostics bundles may contain Personal Data such as usernames and IP address.
+These diagnostic bundles are accessible only to Docker Inc. developer teams in order to investigate related issues.
```

Comment:
The diagnostics bundles are only accessible to a Docker Inc. employees who are directly involved in diagnosing Docker Desktop issues.

Reply:
done